tests/integration/test_engine/test_events/data/test_cases/cases_engine_events.yaml

```diff
@@ -0,0 +1,94 @@
+      - 2019-11-27 15:09:34,716 | INFO  | Apache ActiveMQ 5.15.9 (localhost, ID:5323986a6080-37033-1572227374550-0:1) is shutting down | org.apache.activemq.broker.BrokerService | main
+      - 2019-11-27 15:09:34,718 | INFO  | Connector openwire stopped | org.apache.activemq.broker.TransportConnector | main
+      - 2019-11-27 15:09:34,719 | INFO  | Connector amqp stopped | org.apache.activemq.broker.TransportConnector | main
+      - 2019-11-27 15:09:34,721 | INFO  | Connector stomp stopped | org.apache.activemq.broker.TransportConnector | main
+      - 2019-11-27 15:09:34,722 | INFO  | Connector mqtt stopped | org.apache.activemq.broker.TransportConnector | main
+      - 2019-11-27 15:09:34,723 | INFO  | Connector ws stopped | org.apache.activemq.broker.TransportConnector | main
+      - 2019-11-27 15:09:34,727 | INFO  | PListStore:[/opt/activemq/data/localhost/tmp_storage] stopped | org.apache.activemq.store.wawa.plist.PListStoreImpl | main
+      - 2019-11-27 15:09:34,728 | INFO  | Stopping async queue tasks | org.apache.activemq.store.wawa.KahaDBStore | main
+      - 2019-11-27 15:09:34,730 | INFO  | Stopping async topic tasks | org.apache.activemq.store.wawa.KahaDBStore | main
+      - 2019-11-29 10:59:49,515 | INFO  | No Spring WebApplicationInitializer types detected on classpath | /admin | main
+      - INFO  | anonymous called org.apache.activemq.broker.jmx.QueueView.retryMessages[] at 27-11-2019 08:45:57,213 | qtp443290224-47
+      - INFO  | admin called org.apache.activemq.broker.jmx.QueueView.retryMessages[] at 27-11-2019 08:45:57,229 | qtp443290224-45
```

Comment:
There are some events that you need to check if they should be included.

Reply:
Solved https://github.com/wazuh/wazuh-qa/pull/3809/commits/682e35e6b80b44e47ae0fa8991abc0376745df69